src/legacy/server/saved_objects/service/lib/search_dsl/query_params.test.ts

```diff
@@ -334,7 +335,7 @@ describe('searchDsl/queryParams', () => {
 
   describe('search, searchFields', () => {
     it('includes all types for field', () => {
-      expect(getQueryParams(MAPPINGS, SCHEMA, null, null, 'y*', ['title'])).toEqual({
+      expect(getQueryParams(MAPPINGS, SCHEMA, undefined, undefined, 'y*', ['title'])).toEqual({
```

Comment:
just thinking loudly for the future - when `arguments.length > 3` named arguments are more readable. the same expression:
```js
getQueryParams({
   mapping,
   schema,
   search: 'y*',
   searchFields: [title]
})
```

Reply:
++ I totally agree with this, I have a future plan to do some changes like that to the saved object repository.